### ISSUE TITLE
Unicode character issue

### ISSUE THREAD

**Issue by magyarosibotond** [traduora/traduora#20]
**Describe the bug**
Translation.value does not support unicode characters. It seems like VARCHAR does not support these special characters by default.

**To Reproduce**
1. Add a new translation to the project: Hungarian, Romanian or Arabic.
2. Try to translate a term in one of the mentioned languages.
3. Enter őúüöá and hit save.
4. No error on client side, Query error thrown on backend side.
5. Notice that the value stays on the UI, until the page is refreshed.

**Expected behavior**
The value should be saved to the database.

**Log**
```text
[0] { QueryFailedError: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xC5\xB1\xC3\xA9' for column 'value' at row 1
[0]     at new QueryFailedError (/Users/<User>/Documents/open-source/traduora/api/src/error/QueryFailedError.ts:9:9)
[0]     at Query.<anonymous> (/Users/<User>/Documents/open-source/traduora/api/src/driver/mysql/MysqlQueryRunner.ts:163:37)
[0]     at Query.<anonymous> (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/Connection.js:502:10)
[0]     at Query._callback (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/Connection.js:468:16)
[0]     at Query.Sequence.end (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
[0]     at Query.ErrorPacket (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/protocol/sequences/Query.js:90:8)
[0]     at Protocol._parsePacket (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/protocol/Protocol.js:278:23)
[0]     at Parser.write (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/protocol/Parser.js:76:12)
[0]     at Protocol.write (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/protocol/Protocol.js:38:16)
[0]     at Socket.<anonymous> (/Users/<User>/Documents/open-source/traduora/api/node_modules/mysql/lib/Connection.js:91:28)
[0]   message:
[0]    'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: \'\\xC5\\xB1\\xC3\\xA9\' for column \'value\' at row 1',
[0]   code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
[0]   errno: 1366,
[0]   sqlMessage:
[0]    'Incorrect string value: \'\\xC5\\xB1\\xC3\\xA9\' for column \'value\' at row 1',
[0]   sqlState: 'HY000',
[0]   index: 0,
[0]   sql:
[0]    'INSERT INTO `translation`(`termId`, `projectLocaleId`, `value`, `dateCreated`, `dateModified`) VALUES (\'02016f40-666a-4048-9b73-b5aa45cd3663\', \'b7c304e9-a384-4486-902e-38fdc001115a\', \'űé\', DEFAULT, DEFAULT)',
[0]   name: 'QueryFailedError',
[0]   query:
[0]    'INSERT INTO `translation`(`termId`, `projectLocaleId`, `value`, `dateCreated`, `dateModified`) VALUES (?, ?, ?, DEFAULT, DEFAULT)',
[0]   parameters:
[0]    [ '02016f40-666a-4048-9b73-b5aa45cd3663',
[0]      'b7c304e9-a384-4486-902e-38fdc001115a',
[0]      'űé' ] }
[0] 127.0.0.1 - PATCH /api/v1/projects/744ae9c4-54ee-421d-a135-5e03845ff774/translations/ar_AE HTTP/1.1 500 83 - 282.020 ms
```

**Environment (please complete the following information):**
 - Device: Desktop
 - OS: Mac OS 10.14.3
 - Browser: Safari
 - Version: 12.0.3

**Additional context**
Running `bin/start-dev.sh` with a `mysql/mysql:5.7` container.

**Comment by sodik82** [comment]
change in docker-compose.yml helped:

```
  mysqldb:
    image: mysql:5.7
    command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
```

**Comment by anthonynsimon** [comment]
It's likely that your database is not using utf8 by default. I think the right way to go about this might be to make add a migration to make sure that traduora's table's are utf8 by default. Will investigate which way to go about this.

**Comment by anthonynsimon** [comment]
@sodik82 @magyarosibotond  this should be fixed in #22, let me know what you guys think.

**Comment by magyarosibotond** [comment]
@anthonynsimon 👌can’t wait!
